README.md

```diff
@@ -1 +1,50 @@
-# todo-list
\ No newline at end of file
+# To-do-List
+
+>To-do-List website is a page that contains information that you already listed of shores and things to do .
+
+The pull request is opened to get a review on the project and check for any required changes.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+- Webpack
+
+### Setup
+
+> Open GitHub and go to my GitHub repository name: to do list.to clone.
+> Click “Code” and copy the given URL.
+> click code and copy the URL.
+> click code and copy the URL.
+> Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
+> Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
+> Syntax:
+$ git clone {repository URL}
```

Comment:
- [ ] [OPTIONAL] I Kindly suggest that you add steps to run the project locally. This will help users know how to run your project.

## EXAMPLE
```
### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/anasbassi/todo-list/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/anasbassi/todo-list.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:8080](http://localhost:8080) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```